chain-vote/src/tally.rs

```diff
@@ -647,6 +656,43 @@ mod tests {
         );
     }
 
+    #[test]
+    fn tally_wrong_elements_size() {
+        let mut rng = ChaCha20Rng::from_seed([0u8; 32]);
+
+        let shared_string =
+            b"Example of a shared string. This should be VotePlan.to_id()".to_owned();
+        let h = Crs::from_hash(&shared_string);
+
+        let mc = MemberCommunicationKey::new(&mut rng);
+        let mc = [mc.to_public()];
+
+        let threshold = 1;
+
+        let m = MemberState::new(&mut rng, threshold, &h, &mc, 0);
+
+        let participants = vec![m.public_key()];
+        let ek = ElectionPublicKey::from_participants(&participants);
```

Comment:
But at this case, we will have cyclic package dependency, because `committee_members_manager` defined in the `chain-impl-mockchain` and this package use as a dependency `chain-vote` package